File: Python/Punto_11.py
# Se crean tres vectores de 10 posiciones cada uno
vector1=[0]*10
vector2=[0]*10
vector3=[0]*10
# Se pide al usuario que llene cada vector
vector1=list(map(lambda elemento:int(input("Ingrese un número: ")),vector1))
vector2=list(map(lambda elemento:int(input("Ingrese un número: ")),vector2))
vector3=list(map(lambda elemento:int(input("Ingrese un número: ")),vector3))
# Se crea una matriz de 3 elementos
matriz=[0]*3
# Se llena cada elemento de la matriz con los vectores creados
matriz[0]=vector1
matriz[1]=vector2
matriz[2]=vector3
# Se muestra la matriz creada
print(matriz)

File: Java/Punto11.java
package Java;

import java.util.Arrays;

public class Punto11 {
    public static void main(String[] args) {
        int[] vector1 = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
        int[] vector2 = {11, 12, 13, 14, 15, 16, 17, 18, 19, 20};
        int[] vector3 = {21, 22, 23, 24, 25, 26, 27, 28, 29, 30};

        int[][] matriz = {vector1, vector2, vector3};

        // Imprimir matriz
        Arrays.stream(matriz).forEach(fila -> {
            Arrays.stream(fila).forEach(num -> System.out.print(num + " "));
            System.out.println();
        });
    }
}